py/jupyterlite/src/jupyterlite/app.py

```diff
@@ -80,6 +85,10 @@ class BaseLiteApp(JupyterApp, LiteBuildConfig, DescribedMixin):
 
     flags = lite_flags
 
+    @default("config_file_paths")
+    def _config_file_paths_default(self):
+        return [str(Path.cwd())] + jupyter_config_path()
```

Comment:
yep, let's get rid of cwd.

Reply:
it'll probably break some tests/binder, though.

Reply:
ok let's do the change in its own PR so it's more visible in the changelog.

I guess that means `--config jupyter_lite_config.json` should now be used explicitly.

Reply:
ugh. a good point. i guess we need to think about it a bit more: running `jupyter lite build` from a CLI is a different activity altogether than browsing a web page.